src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.ts

```diff
@@ -32,6 +33,9 @@ export class TutorialGroupDetailComponent implements OnChanges {
     formattedAdditionalInformation?: SafeHtml;
     getDayTranslationKey = getDayTranslationKey;
 
+    faQuestionCircle = faQuestionCircle;
+    math = Math;
```

Comment:
same for the other classes
```suggestion
    readonly Math = Math;
```

Reply:
Done in 5a44a6571d5bb5b56e26be52ddc3fb6e25c3276e